static/js/controllers/reports-add.js

```diff
@@ -19,9 +20,7 @@ angular.module('inboxControllers').controller('ReportsAddCtrl',
         return $q.resolve({ form: $state.params.formId });
       }
       if ($state.params.reportId) { // editing
-        return DB().get($state.params.reportId, {
-          attachments: true
```

Comment:
It was good, but it meant we had to manually access the _attachments and convert to string. This way we can use PouchDBs getAttachment function which handles that for us.